frontend/packages/kubevirt-plugin/src/components/modals/support-modal/support-modal.tsx

```diff
@@ -47,6 +54,18 @@ const SupportModal: React.FC<SupportModalProps> = ({ onConfirm, close, community
                 />
               </StackItem>
             </>
+          ) : isCommonTemplate ? (
+            <>
+              <StackItem>
+                {t('kubevirt-plugin~This template is provided by Red Hat, But is not supported')}
+              </StackItem>
+              <StackItem>
```

Comment:
the `Lern more` external link can be part of the same `<StackItem>` as the info text